site/_apps/angular.txt

```diff
@@ -0,0 +1,14 @@
+---
+weight: 5
+title:  Angular
+logo: /assets/images/angular-logo.svg
+lighthouse: 100/100
+interactive-em: 5.8s
+interactive-faster-3g: 3.48s
+lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170423_BN_32H&run=1
+wpt-em-link: https://www.webpagetest.org/result/170420_46_67R/
+wpt-faster-3g-link: https://www.webpagetest.org/result/170328_N6_761d4f12fce28adc5163a3d056ce0af6/
+image: /assets/images/angular2hn-mobile.png
```

Comment:
You're right, they're all quite large! used `compressor` to trim them down (for some of them ~70% reduction in size :O)

<img width="302" alt="screen shot 2017-04-23 at 11 05 04 pm" src="https://cloud.githubusercontent.com/assets/12476932/25321054/9f959e24-2879-11e7-8ec8-3df7953b4ff8.png">